internal/resources/standalone/bindata.go

```diff
@@ -329,7 +329,7 @@ var _bintree = &bintree{nil, map[string]*bintree{
 	"grpc-bg.svg":              {grpcBgSvg, map[string]*bintree{}},
 	"grpc-logo.png":            {grpcLogoPng, map[string]*bintree{}},
 	"index-template.html":      {indexTemplateHtml, map[string]*bintree{}},
-	"jquery-3.4.1.min.js":      {jquery341MinJs, map[string]*bintree{}},
+	"jquery-3.6.0.min.js":      {jquery341MinJs, map[string]*bintree{}},
```

Comment:
unfamiliar with this code, but: this line still says `jquery341MinJs`?

Reply:
changed :)